### DISCUSSION TITLE
Native Typescript

### DISCUSSION THREAD

**Discussion by bevand10** [Envek/aws-sam-typescript-layers-example#2] [General]
Feels like a missed opportunity to explore compile-less Typescript deployment to Lambda by exploiting the newish native TS runtime - Deno - built by the creator of NodeJS, Ryan Dahl?

https://deno.land/

**Comment by Envek** [comment]
It will require [creating a brand new runtime](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/building-custom-runtimes.html) instead of using `nodejs` runtime provided by AWS.

It worth researching, but it will be totally different solution.

**Reply by bevand10** [reply]
It exists, and has SAM:

https://deno.land/x/lambda@1.7.2
